lua/cstm/lsp/server/gopls.lua

```diff
@@ -1,8 +1,12 @@
 local nvim_lsp = require('lspconfig')
+local go_code_actions = require('cstm.lsp.handlers.code_action').go_code_actions
 
 local setup = function(on_attach)
   nvim_lsp.gopls.setup({
     on_attach = on_attach,
+    handlers = {
+      ['textDocument/codeAction'] = go_code_actions,
```

Comment:
Add comment mentioning that this is WIP, or entirely remove this and add the custom handlers to the list in `lsp/init.lua`

https://github.com/aliou/dotvim/blob/0dd8c651a4268f6bf885732fd234360cd2ec4332/lua/cstm/lsp/init.lua#L44-L49